feature/gribi/otg_tests/gribi_scaling/README.md

```diff
@@ -13,29 +13,24 @@ Validate gRIBI scaling requirements.
 *   On DUT, create a policy-based forwarding rule to redirect all traffic received from DUT port-1 into VRF-1 (based on src. IP match criteria)
 *   Establish gRIBI client connection with DUT negotiating FIBACK as the
     requested ack_type and make it become leader.
-*   Using gRIBI Modify RPC install the following IPv4Entry sets, and validate
+*   TODO: Using gRIBI Modify RPC install the following IPv4Entry sets, and validate
     the specified behaviours:
-    *   <Default VRF> IPv4Entries -> NHG -> Multiple NH.
-        *   Inject IPv4Entries(IPBlockDefaultVRF: 198.18.196.1/22) in default
-            VRF
-        *   Install 64 L3 sub-interfaces IP to NextHopGroup containing one
-            NextHop specified to ATE port-2.
-        *   Validate that the entries are installed as FIB_PROGRAMMED
-    *   <VRF1> IPv4Entries -> Multiple NHG -> Multiple NH.
-        *   Inject IPv4Entries(IPBlock1: "198.18.0.1/18") in VRF1.
-        *   Install 1000 IPs from IPBlockDefaultVRF to 10 NextHopGroups
-            containing 100 NextHops each
-        *   Validate that the entries are installed as FIB_PROGRAMMED
-    *   <VRF2> IPv4Entries -> Multiple NHG -> Multiple NH.
-        *   Inject IPv4Entries(IPBlock2: "198.18.64.1/18") in VRF2.
-        *   Install *repeat* 17.5K NH from 1K /32 from IPBlockDefaultVRF to 35
-            NextHopGroups containing 45 NextHops each
-        *   Validate that the entries are installed as FIB_PROGRAMMED
-    *   <VRF3> IPv4Entries -> Multiple NHG -> Multiple NH.
-        *   Inject IPv4Entries(IPBlock3: "198.18.128.1/18") in VRF3.
-        *   Install IPiniP decap-then-encap to 500 first /32 from <IPBlockVRF1>
-            to 500 NextHopGroups containing 1 NextHop each
-        *   Validate that the entries are installed as FIB_PROGRAMMED
+    *   <Default VRF>
+        * A) Install 400 NextHops, egressing out different interfaces.
+        * B) Install 200 NextHopGroups.  Each points at 8 NextHops from A).
+        * C) Install 200 IPv4 Entries, each pointing at a unique NHG (1:1) from B.
+        * D) Install 200 NextHops.  Each will redirect to an IP from C).
+        * E) Install 100 NextHopGroups.  Each will contain 2 NextHops from D). The backup next_hop_group will be to redirect to VRF2.
+        * F) Install 100 NextHopGroups.  Each will contain 2 NextHops from D). The backup next_hop_group will be to decap and redirect to DEFAULT vrf.
+        * G) Install 700 NextHops.  Each will decaps + reencap to an IP in VRF2.
+        * H) Install 700 NextHopGroups.  Each will point to a NextHop from G) and have a backup next_hop_group to decap and redirect to DEFAULT vrf.
+    *   <VRF1>
+        *   Install 9000 IPv4Entries.  Each points to a NextHopGroup from E).
+    *   <VRF2>
+        *   Install 9000 IPv4Entries (Same IPAddress as VRF1).  Each points to a NextHopGroup from F).
```

Comment:
if we are using same IP address as VRF1 means , are we not planning to same destination ip as received  in encapped packet?

Reply:
Updated the VRF procedure - there should be different bheaviour based on the SRC ip.  Let me know if this is unclear.

Reply:
sure that is correct if we directly hit VRF2  through PBF match . but if we want to validate VRF1 backup path and make sure that really dst_ip and src_ip has been updated as per encap then we need to have different prefixes in VRF2.

Reply:
In this test I don't think we want to validate the repaired path; we have more-specific tests for that.